<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToSafeSettings</class>
 <widget class="QDialog" name="ToSafeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="question_layout" stretch="3,7">
    <item>
     <widget class="QLabel" name="ques_lb">
      <property name="text">
       <string>密保问题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ques_le">
      <property name="text">
       <string>这里放从数据库里拿出来的密保问题</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ans_layout" stretch="3,7">
    <item>
     <widget class="QLabel" name="ans_lb">
      <property name="text">
       <string>密保答案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ans_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
